src/Commands/GenerateTypesCommand.php

```diff
@@ -93,6 +98,7 @@ public function handle(): void
     {
         $this->modelDir = $this->option('modelDir') ?? config('laravel-model-ts-type.model_dir');
         $this->outputDir = $this->option('outputDir') ?? config('laravel-model-ts-type.output_dir');
+        $this->useKebabCase = !($this->option('noKebabCase') ?? config('laravel-model-ts-type.no_kebab_case'));
```

Comment:
This is a good addition, we didn't consider this. Could you also add this option to the default configfile `src/config/laravel-model-ts-type.php` and the command options in the README?

Reply:
@RicoClark thanks for the review! 😊  I just pushed a few changes. Could you please check it again?